HouseRules_Configuration/ConfigManager.cs

```diff
@@ -105,22 +105,44 @@ internal string ExportRuleset(Ruleset ruleset)
         }
 
         /// <summary>
-        /// Imports all rulesets in the default ruleset directory.
+        /// Tries to import all rulesets in the default ruleset directory.
         /// </summary>
-        /// <returns>The list of imported rulesets.</returns>
-        internal List<Ruleset> ImportRulesets()
+        /// <param name="rulesets">The rulesets that were successfully imported.</param>
+        /// <returns>Whether or not issues were encountered when importing rulesets.</returns>
+        internal bool TryImportRulesets(out List<Ruleset> rulesets)
         {
             var files = Directory.EnumerateFiles(RulesetDirectory, "*.json").ToList();
             ConfigurationMod.Logger.Msg($"Found [{files.Count}] rulesets in directory [{RulesetDirectory}]");
-            return files.Select(ImportRuleset).ToList();
+
+            var hadNoErrors = true;
+            rulesets = new List<Ruleset>();
+            foreach (var file in files)
+            {
+                try
+                {
+                    rulesets.Add(ImportRuleset(file, tolerateFailures: false));
+                }
+                catch (Exception e)
+                {
+                    hadNoErrors = false;
+                    ConfigurationMod.Logger.Warning($"Failed to import ruleset from file [{file}]. Skipping that ruleset: {e}");
+                }
+            }
+
+            return hadNoErrors;
```

Comment:
Was wondering about returning a list of the failed [{file}]s here, so they could be displayed in-game.

Reply:
Done.  Moved implementation around so that now a list of files that failed to be loaded is captured.